cloud-foundry/nginx/conf/includes/redirect.conf

```diff
@@ -2,24 +2,24 @@ rewrite ^/task-maven-latest$ https://repo.spring.io/libs-release/org/springframe
 rewrite ^/task-docker-latest$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.SR2/spring-cloud-task-app-descriptor-Elston.SR2.task-apps-docker redirect;
 
 rewrite ^/Elston-GA-task-applications-maven$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-maven redirect;
-rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-maven redirect;
+rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/snapshot/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-maven redirect;
 rewrite ^/Elston-GA-task-applications-docker$ https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.RELEASE/spring-cloud-task-app-descriptor-Elston.RELEASE.task-apps-docker redirect;
-rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-docker$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-docker redirect;
+rewrite ^/Elston-BUILD-SNAPSHOT-task-applications-docker$ https://repo.spring.io/snapshot/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Elston.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Elston.BUILD-SNAPSHOT.task-apps-docker redirect;
 
 rewrite ^/Dearborn-SR1-task-applications-maven$ https://repo.spring.io/libs-release-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.SR1/spring-cloud-task-app-descriptor-Dearborn.SR1.task-apps-maven redirect;
-rewrite ^/Dearborn-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/libs-snapshot-local/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Dearborn.BUILD-SNAPSHOT.task-apps-maven redirect;
+rewrite ^/Dearborn-BUILD-SNAPSHOT-task-applications-maven$ https://repo.spring.io/snapshot/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Dearborn.BUILD-SNAPSHOT/spring-cloud-task-app-descriptor-Dearborn.BUILD-SNAPSHOT.task-apps-maven redirect;
```

Comment:
Is this because the new snapshots won't be published into `libs-snapshot-local` but will only be at `snapshot`?